src/runtime/audience-action-local-flow.ts

```diff
@@ -647,15 +701,22 @@ export class AudienceActionLocalFlow implements AudienceActionFlow {
   }
 
   private getCloseButtonHtml_(html: string) {
```

Comment:
nit: the function name seems misleading that it can return an empty string. I think ideally the caller should check the condition, but I guess we can simply correct it by renaming the function like `getCloseButtonOrEmptyHtml_`?

Reply:
Done